WcaOnRails/app/models/user.rb

```diff
@@ -727,24 +727,26 @@ def serializable_hash(options = nil)
 
   def to_wcif(competition, registration = nil, registrant_id = nil)
     person_pb = [person&.ranksAverage, person&.ranksSingle].compact.flatten
+    roles = registration&.roles || []
+    roles << "delegate" if competition.delegates.include?(self)
+    roles << "organizer" if competition.organizers.include?(self)
     {
       "name" => name,
       "wcaUserId" => id,
       "wcaId" => wca_id,
       "registrantId" => registrant_id,
       "countryIso2" => country_iso2,
-      "delegatesCompetition" => competition.delegates.include?(self),
-      "organizesCompetition" => competition.organizers.include?(self),
```

Comment:
Do we know of any software using this already?

Reply:
CubingUSA does: https://github.com/cubingusa/org/blob/3441353f06f2a1eb85faaa80fefd92acc3d43b64/src/scheduling/wcif/person.py#L53

But it would be easy enough to switch to using the new form.

Reply:
Nah, feel free to just delete it, just please let me know when you push to prod and I'll update.

I'm not worried about a short-term breakage, since it's pretty much just us nats that uses it now.

Reply:
@timreyn I've just recalled that I should've let you know once this gets merged, so doing it now =)